src/server.js

```diff
@@ -27,15 +27,9 @@ app.use('/public', express.static(path.join(__dirname, '/public')));
 // Serve the Parse API on the /parse URL prefix
 app.use(PARSE_MOUNT || '/parse', api);
 
-// Parse Server plays nicely with the rest of your web routes
+// health endpoint
 app.get('/', async (request, response) =>
```

Comment:
What is this?

Reply:
I removed the example endpoint and instead created an endpoint you could query to check if the server is online.